x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/description_step/index.tsx

```diff
@@ -200,11 +201,14 @@ export const getDescriptionItem = (
     const query = get('queryBar.query.query', data);
     const savedId = get('queryBar.saved_id', data);
     const savedQueryName = get('queryBar.title', data);
+    const ruleType: Type = get('ruleType', data);
+    const queryLabel = getQueryLabel(ruleType);
     return buildQueryBarDescription({
```

Comment:
would be nice to consider adding an integration test for `getDescriptionItem` where we check query label is applied from `getQueryLabel` function.

There is already some tests for this in `x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/description_step/index.test.tsx`, so should be simple to add one test for it

Reply:
Does something like this cover what you are thinking? 

[`e92b0dd` (#178821)](https://github.com/elastic/kibana/pull/178821/commits/e92b0dd428940377735b7c1d61173ca8c71570aa)